scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala

```diff
@@ -47,15 +48,15 @@ trait TypedSeperatedFile extends Serializable {
 /**
  * Typed tab separated values file
  */
-object TypedTsv extends TypedSeperatedFile {
```

Comment:
Can we instead just override the methods in TypedSeparatedFile for maximum source code compatibility?

I think we could pretty easily implement the 4 apply methods there, then we are fairly sure this change won't break anyone, which is nice.